src/libraries/System.Text.RegularExpressions/gen/RegexGenerator.Emitter.cs

```diff
@@ -1599,7 +1562,7 @@ void EmitWhenHasCapture()
                             writer.WriteLine($"matchIndex = base.MatchIndex({capnum});");
                             using (EmitBlock(writer, $"for (int i = 0; i < matchLength; i++)"))
                             {
-                                using (EmitBlock(writer, $"if ({ToLower(hasTextInfo, options, $"inputSpan[matchIndex + i]")} != {ToLower(hasTextInfo, options, $"{sliceSpan}[i]")})"))
+                                using (EmitBlock(writer, $"if ({ToLower($"inputSpan[matchIndex + i]")} != {ToLower($"{sliceSpan}[i]")})"))
```

Comment:
This is now the only place that uses ToLower, right?  I'd suggest then it just be:
```suggestion
                                using (EmitBlock(writer, $"if (_textInfo.ToLower(inputSpan[matchIndex + i]) != _textInfo.ToLower({sliceSpan}[i]))"))
```